arch/nds/Makefile.in

```diff
@@ -50,6 +47,10 @@ arch/nds/%.o: arch/nds/%.c
 	$(if ${V},,@echo "  CC      " $<)
 	${CC} -MD ${core_cflags} ${core_flags} -Wno-unused-macros -c $< -o $@
 
+arch/nds/%.bin.o arch/nds/%_bin.h : arch/nds/%.bin
+	$(if ${V},,@echo "  BIN2O   " $<)
+	bin2s -a 32 -H arch/nds/`(echo $(<F) | tr . _)`.h $< | $(AS) -o arch/nds/$(<F).o
+
```

Comment:
This doesn't work anymore. See 1bae72a8 and the corresponding fix I added to the 3DS Makefile in 234ddbf6.